src/LibraryMangementSystem/Author.java

```diff
@@ -0,0 +1,27 @@
+package LibraryMangementSystem;
+
+import java.util.Date;
+import java.util.List;
+
+public class Author extends Member {
+
+    private String authorNationality;
+    private Date dateOfBirth;
+    private List<Book> authorBookList;
+
+    public String displayAuthorInformation() {
+        return getName() + " has the " + authorNationality + " nationality and was born in " + dateOfBirth;
+    }
+
+    public void addBooks(Book book) {
+        if (authorBookList.contains(book)) {
+            System.out.println("This book's author is already: " + getName());
```

Comment:
- maybe, the method should be called at singular , because we only add one book?